src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -571,13 +512,99 @@ def get_worker_id(self) -> str:
 
 
 @dataclass
-class PosixInstanceWorker(EC2InstanceWorker):
+class WindowsInstanceBuildWorker(WindowsInstanceWorkerBase):
     """
-    This class represents a Linux EC2 Worker Host.
-    Any commands must be written in Bash.
+    This class represents a Windows EC2 Worker Host.
+    Any commands must be written in Powershell.
     """
 
-    AL2023_AMI_NAME: ClassVar[str] = "al2023-ami-kernel-6.1-x86_64"
+    WIN2022_AMI_NAME: ClassVar[str] = "Windows_Server-2022-English-Full-Base"
+
+    def configure_worker_command(self, *, config: DeadlineWorkerConfiguration) -> str:
+        """Get the command to configure the Worker. This must be run as Administrator."""
+        cmds = [
+            "Set-PSDebug -trace 1",
```

Comment:
The debug tracing is an intentional addition. If this ssm command fails, then it's challenging to understand why without it.